Scribble/Scribble/ColorViewController.swift

```diff
@@ -87,17 +87,17 @@ class ColorViewController: UIViewController {
         }
     }
     
-    func selectButton(button: UIButton?) {
+    func selectButton(_ button: UIButton?) {
```

Comment:
There is new requirement in swift 3 that parameters in a function to have a label the underscore just says allow this to be called without a label. This was the result of the automatic conversion to swift 3 from swift 2.

There is stackoverflow question that kind of explains this new syntax http://stackoverflow.com/questions/39627106/why-do-i-need-underscores-in-swift

Seems like an odd change to me from swift2 to swift3